script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1_1.txt

```diff
@@ -22,21 +22,21 @@ I resist the urge to say, "No, seven o'clock is normal. This house is not normal
 <Page8>
 "Hmm. What time do you get up, Akiha?"
 "I wake up at five o'clock, what about it?"
-"...I'm sorry, My mistake."
+"...I'm sorry, my mistake."
 Despite saying something that would be unthinkable for a normal person, she is wearing an expression that says "is that so strange?". I can't help but be dumbfounded.
-I've never seen a more perfect response from her.
+I've never heard such a perfect response.
 <Page9>
-"? What did you mean by "your mistake"?"
-"No, nothing...but what are you doing up so early? Trying not to be late?"
+"What do you mean your 'mistake'?"
+"Ah... Nevermind that, what are you doing up so early? Trying not to be late?"
```

Comment:
this is a little stiff, so I'm open to comments.

Reply:
Reads ok to me